tutorials/fileIO/src/main/java/com/example/fileIO/BytesFileWriter.java

```diff
@@ -0,0 +1,100 @@
+package com.example.fileIO;
+
+import java.util.ArrayList;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.datatorrent.api.DefaultInputPort;
+import com.datatorrent.lib.io.fs.AbstractFileOutputOperator;
+
+public class BytesFileWriter extends AbstractFileOutputOperator<byte[]>
+{
```

Comment:
Extend BytesFileWriter from GenericFileOutputOperator.BytesFileOutputOperator instead of extending AbstractFileOutputOperator.